qutebrowser/utils/utils.py

```diff
@@ -889,3 +889,25 @@ def cleanup_file(filepath: str) -> Iterator[None]:
             os.remove(filepath)
         except OSError as e:
             log.misc.error(f"Failed to delete tempfile {filepath} ({e})!")
+
+
+def parse_int_set(s: str, universe: Iterable[int]) -> Iterable[int]:
+    """Parse a comma-separated list of numbers and range specifications.
+
+    The wildcard * is also allowed, in which case the `universe` is returned.
```

Comment:
I think it might make sense to also allow something like `last`, so that you could do `1-last` (all tabs), `5-last`, etc.

See #2493 (which was abandoned) for some inspiration.